js/src/forum/ForumApplication.js

```diff
@@ -63,6 +64,13 @@ export default class ForumApplication extends Application {
    */
   history = new History();
 
+  /**
+   * @inheritdoc
+   */
+  cache = {
+    discussionList: new DiscussionListState(),
+  };
```

Comment:
Sorry if this was discussed elsewhere, I did not find anything about it in #2144... Why is this nested under the `cache`, unlike the various other new state objects?

Reply:
Exclusively for BC reasons, to not break stuff that calls app.cache.discussionList. Although tbh, we could very well store it as a direct property of app, and have app.cache.discussionList just be a reference.

Reply:
Okay, I guess the other changes made this more clear - you probably want to reduce breakage, because other places were already using this... I guess it's still broken for most usages, though, so renaming might be an option for consistency? Or is there another semantic difference you see between the other states and the "cache"?

Reply:
On the bright side, it isn't really broken because all the public api methods from the component exist on the state. Moved, BC layer added.